src/verify.c

```diff
@@ -667,13 +669,19 @@ int verify_main(int argc, char **argv)
 	times = init_timelist();
 
 	grabtime_start(&times, "Load and recurse Manifests");
+load_manifests:
 	official_manifest = load_mom(version);
-
 	if (!official_manifest) {
 		/* This is hit when or if an OS version is specified for --fix which
 		 * is not available, or if there is a server error and a manifest is
 		 * not provided.
 		 */
+		if (retries < MAX_TRIES) {
+			increment_retries(&retries, &timeout);
+			printf("Retry #%d downloading from/to MoM Manifests\n", retries);
+			goto load_manifests;
+		}
```

Comment:
Sure, then I think load_mom should have these multiple retries when downloading and loading it up, and then a single cleanup/retry if signature fails.